web_widget_plotly_chart/readme/INSTALL.rst

```diff
@@ -0,0 +1,3 @@
+You need to install the python plotly library::
+
+    pip3 install plotly==4.1.0
```

Comment:
Why pin the version in the description and not in the requirement file and manifest ?

Reply:
Forgot that! I'm updating it right now